spice_ev/costs.py

```diff
@@ -149,10 +146,10 @@ def calculate_costs(strategy, voltage_level, interval,
     :type price_list: list
     :param power_fix_load_list: power supplied from the grid for the fixed load
     :type power_fix_load_list: list
+    :param power_schedule_list: power to be supplied or fed-in according to schedule
+    :type power_schedule_list: list
```

Comment:
is this argument really required or can it default to None (in which case no schedule costs should be computed)? This would simplify the example files and facilitate backwards compatibility with other projects that use the cost calculation, like the eBus-Toolbox.

Reply:
For the schedule strategy it is required. Calculating costs for this strategy without having a schedule does not really make sense. But of course there is the possibility to set the deviation costs to zero in case of a missing schedule and print a warning. What do you think?

Reply:
My point is that it is not required for any strategy other than schedule. Therefore, I would let it default to None and add a check in the schedule-cost-calculation that it is set (debatable what happens when this check fails, stopping the cost calculation or just printing a warning and continuing are both fine by me, although I prefer the first).

Reply:
If we run several simulations in a row and also run the cost calculation via the simulate.py but stop the cost calculation when no schedule is given even though the calculation is executed for the schedule strategy, the whole simulation will stop. I am not sure if this should happen. But this is why I would prefer a warning. @jakobgemassmer what do you think about this?

Reply:
Setting it to `None` by default would be good (since this function wouldn't work without a `power_schedule_list` (other strategy chosen), if I understand correct).
I would also prefer a warning for the same reasons as you mentioned, Tabea.